docs/source/admin_guide/backup.md

```diff
@@ -145,9 +147,27 @@ tar -xvf 2021-04-23.tar
 
 The file permissions for the default tar is same as the original files.
 
+> **Important: If upgrading from 0.3.14 or earlier to 0.4 or later**
+>
+> QHub v0.4: If restoring your NFS as part of the upgrade you must also run some extra commands, immediately after extracting from the tar file.
+>
+> Because previous versions contained the `shared` folder within `home`, but they are now both at the same level, you must run:
+> ```shell
+> cd /data
+> cp -r home/shared/* shared/
+> rm -rf home/shared/
+> ```
+> And then:
+> ```shell
+> chown -R 1000:100 /data/home/*
+> chown -R 1000:100 /data/shared/*
+> ```
+> This is because all users have the same uid in QHub v0.4 onward.
+>
+
 ### Google cloud provider
 
-To use the Google Cloud provider, install [gsutil](https://cloud.google.com/storage/docs/gsutil_install). The instructions are same as the preceding steps. Additionally, use these commands for copy/download of the backup:
+To use the Google Cloud provider, install the [gsutil](https://cloud.google.com/storage/docs/gsutil_install) CLI. The instructions are the same as the preceding steps. Additionally, use these commands for copy/download of the backup:
```

Comment:
Again what are the preceding steps? do you refer to AWS? if so make it explicit otherwise folks will get lost